.github/stale.yml

```diff
@@ -1,5 +1,5 @@
 # Number of days of inactivity before an Issue or Pull Request becomes stale
```

Comment:
If we are using `.github/workflows/stale.yml` for handling stale? Is `.github/stale.yml` still needed?

Reply:
Yes b/c as I mentioned in OP the Action can't yet remove the label (like when a new comment is created)

Reply:
> Yes b/c as I mentioned in OP the Action can't yet remove the label (like when a new comment is created)

Could you add a comment in `.github/stale.yml` to explain this (the reason we are keeping the file) ?